dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/multimodule/tests/TestMultiModuleClassLoading.java

```diff
@@ -46,10 +46,12 @@ public class TestMultiModuleClassLoading extends FATServletClient {
     @Server(SERVER_NAME)
     public static LibertyServer server;
 
-    //run against both EE9, EE8 and EE7 features
+    //run against EE10, EE9, EE8 and EE7 features
     @ClassRule
-    public static RepeatTests r = RepeatFaultTolerance.repeat(SERVER_NAME, TestMode.LITE, MicroProfileActions.MP50, MicroProfileActions.MP13,
-                                                              RepeatFaultTolerance.MP21_METRICS20);
+    public static RepeatTests r = RepeatFaultTolerance.repeat(SERVER_NAME, TestMode.LITE, MicroProfileActions.MP60,
```

Comment:
are you sure that all 4 repeats are needed in LITE?

Reply:
Spent a while looking at why this might be before I realised that the whole test has `@Mode(FULL)`.

I'll remove `LITE` here since it has no effect and is confusing.

Reply:
Done